src/data/future-events.yaml

```
@@ -1,5 +1,5 @@
 - city: Hamburg
-  date: '2020-12-31'
+  date: '2021-12-11'
   where: Online
```

I think we should change this as well?

```suggestion
  where: Gasstraße 6A, 22761 Hamburg
```